scripts/validate_docstrings.py

```diff
@@ -543,8 +543,8 @@ def validate_pep8(self):
         application = flake8.main.application.Application()
         application.initialize(["--quiet"])
 
-        with tempfile.NamedTemporaryFile(mode='w') as file:
-            file.write(content)
+        with tempfile.NamedTemporaryFile(mode='wb') as file:
```

Comment:
I'd prefer opening the `NamedTemporaryFile` with `encoding='utf-8'`.

Reply:
Can you address comment from @TomAugspurger here?

Reply:
I changed it as @TomAugspurger suggested.